NOTICE.txt

```diff
@@ -0,0 +1,5 @@
+Copyright (c) 2006-2022 the original author or authors.
```

Comment:
Probably `2023` already.
May I know, please, from what project did you take this notice?
I see that in my other one it is slightly different.
Thanks

Reply:
I tooked the NOTICE File from the [spring-boot](https://github.com/spring-projects/spring-boot/blob/main/buildSrc/src/main/resources/NOTICE.txt) project.